go/service/main.go

```diff
@@ -554,6 +554,9 @@ func (d *Service) hourlyChecks() {
 			d.G().Log.Debug("+ hourly check loop")
 			d.G().Log.Debug("| checking tracks on an hour timer")
 			libkb.CheckTracking(d.G())
+			d.G().Log.Debug("| checking if ephemeral keys need to be created or deleted")
+			ekLib := d.G().GetEKLib()
+			ekLib.KeygenIfNeeded(context.Background())
```

Comment:
It looks like this happens an hour after startup. Should we be more aggressive?

Reply:
yeah i'll call it again above so it kicks off when the service starts (as long as the user is logged in)